models/Address.php

```diff
@@ -60,7 +60,7 @@ public function beforeValidate()
 
     public function getNameAttribute()
     {
-        return $this->getOriginal('name') ?: optional($this->customer)->name;
+        return $this->getRawOriginal('name') ?: optional($this->customer)->name;
```

Comment:
This will break on Laravel prior to v7.

It's been addresses by this commit: https://github.com/OFFLINE-GmbH/oc-mall-plugin/pull/931/commits/2743bdcee4e44d1b3bb0058da6006e121819dc26

Reply:
Thanks!